sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -705,10 +705,10 @@ private void acquireChannel(final ChannelPromiseWithExpiryTime promise) {
 
             if (this.allowedToOpenNewChannel(allowedMaxChannels)) {
                 if (this.connecting.compareAndSet(false, true)) {
-
                     // Fulfill this request with a new channel, assuming we can connect one
                     // If our connection attempt fails, notifyChannelConnect will call us again
 
+                    logger.info("ATTEMPT_TO_CREATE_NEW_CHANNEL");
```

Comment:
debug level please - for most scenarios you wouldn't want/need this in production - info is too verbose here.

Reply:
should probably also include this.endpoint.toString - or this.remoteAddress.toString